_crc32c.c

```diff
@@ -116,8 +116,12 @@ MOD_INIT(crc32c)
 	}
 
 	else {
-		PyErr_SetString(PyExc_ImportError, import_error_msg);
-		return MOD_VAL(NULL);
+#if PY_MAJOR_VERSION >= 3
+		PyRun_SimpleString("print('SSE4.2 extensions providing a crc32c hardware instruction are NOT available in your processor.  A software implementation of the same has been loaded instead.')");
```

Comment:
I think a more precise error message can be stated. Something like:
"crc32c hardware is either unavailable or inaccessible from this library today. A software implementation will be used instead."